doc/funding/process.md

```diff
@@ -4,66 +4,77 @@ This describes the process that the board and the Jupyter community follow in or
 
 Submissions are accepted on a rolling basis throughout the year, with funds being released according to our [budget allocations](./budget.md). 
```

Comment:
```suggestion
```

I think rely on the text below, i.e., we're doing a round, learning from it, and there will be more.